api/src/models/card.schema.ts

```diff
@@ -0,0 +1,13 @@
+import * as mongoose from 'mongoose';
+
+export const CardSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: [true, 'card must have a title']
+  },
+  description: {
+    type: String,
+    required: [true, 'You should describe your service'],
+    minLength: 20,
+  }
```

Comment:
how about the link?
![Screenshot from 2020-09-25 22-32-44](https://user-images.githubusercontent.com/1198975/94313886-d3f1f380-ff7f-11ea-8e82-ae2b46500958.png)